nvflare/private/fed/client/admin_commands.py

```diff
@@ -246,6 +248,34 @@ def process(self, data: Shareable, fl_ctx: FLContext):
         return None
 
 
+class ConfigureJobLogCommand(CommandProcessor):
+    """To implement the configure_job_log command."""
+
+    def get_command_name(self) -> str:
+        """To get the command name.
+
+        Returns: AdminCommandNames.CONFIGURE_JOB_LOG
+
+        """
+        return AdminCommandNames.CONFIGURE_JOB_LOG
+
+    def process(self, data: Shareable, fl_ctx: FLContext):
+        """Called to process the configure_job_log command.
+
+        Args:
+            data: process data
+            fl_ctx: FLContext
+
+        Returns: configure_job_log command message
+
+        """
+        engine = fl_ctx.get_engine()
+        try:
+            dynamic_log_config(data, engine.get_workspace(), fl_ctx.get_job_id())
+        except Exception as e:
+            return secure_format_exception(e)
```

Comment:
when exception happen, this error message is returned, where in the code do we use this?
Do we print it to the users?

Reply:
`err = engine.configure_job_log(job_id, req.body)`
https://github.com/NVIDIA/NVFlare/pull/3127/files#diff-83663edc18c36c1efa66df4048f1f68d57a8f248a4227d82a905d10b958b921cR169

The error message does get added to the conn, else a success message is appended